.github/workflows/test-docs.yaml

```diff
@@ -39,3 +40,12 @@ jobs:
       - name: Build docs / run examples
         run: |
           SPHINXCACHE=${HOME}/.cache/sphinx SPHINXOPTS="-W -j auto" make -C doc html
+
+      - name: "Job has failed: reporting"
+        if: ${{ failure() && github.ref == 'refs/heads/main' }}
+        uses: JasonEtco/create-an-issue@v2
```

Comment:
Should probably pin to commit sha since this has the somewhat sensitive `issues: write` set?